vms/event/services.py

```diff
@@ -1,18 +1,30 @@
 from django.core.exceptions import ObjectDoesNotExist
 from event.models import Event
 
+#check if the event exists and is not empty
+def event_not_empty(event_id):
+    result = True
+    event = get_event_by_id(event_id)
+    if not event:
+        result = False
+    return result
+
 #need to check that this event is not accociated with any jobs, 
```

Comment:
@vubo For now and likely the future, we will only use the docstrings for documentation. We will not use it for the testing function mentioned in your quote above. I'm not a huge fan of doctests ;)